Tasks/Maven/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -23,5 +24,17 @@
   "loc.input.label.jdkUserInputPath": "JDK Path",
   "loc.input.help.jdkUserInputPath": "Sets JAVA_HOME to the given path.",
   "loc.input.label.jdkArchitecture": "JDK Architecture",
-  "loc.input.help.jdkArchitecture": "Optionally supply the architecture (x86, x64) of JDK."
+  "loc.input.help.jdkArchitecture": "Optionally supply the architecture (x86, x64) of JDK.",
+  "loc.input.label.sqAnalysisEnabled": "Run SonarQube Analysis",
+  "loc.input.help.sqAnalysisEnabled": "Run a [SonarQube](http://www.sonarqube.org/) analysis after executing the current goals. 'install' or 'package' goals should be executed first.",
+  "loc.input.label.sqConnectedServiceName": "SonarQube Endpoint",
+  "loc.input.help.sqConnectedServiceName": "The SonarQube server generic endpoint",
```

Comment:
"The endpoint that specifies the SonarQube server to use" might be better (the current text doesn't add much to what's in the label).